moped-editor/src/components/GridTable/GridTableFiltersCommonOperators.js

```diff
@@ -140,4 +140,18 @@ export const GridTableFiltersCommonOperators = {
     envelope: null,
     type: "date",
   },
+  string_equals_case_sensitive: {
+    operator: "_eq",
+    label: "is",
+    description: "Field content equals string (case-sensitive)",
+    envelope: null,
+    type: "string",
+  },
+    string_does_not_equal_case_sensitive: {
+    operator: "_neq",
+    label: "is not",
+    description: "Field content does not equal string (case-sensitive)",
+    envelope: null,
+    type: "string",
+  },
```

Comment:
So I think I must have been misusing `_ilike`, maybe I had the envelope wrong? Because i was trying to get "construction" and not "post-construction" in my searches, but I kept getting containing results and not equality.

Reply:
https://github.com/cityofaustin/atd-moped/blob/main/moped-editor/src/components/GridTable/GridTableFiltersCommonOperators.js#L12

Reply:
I couldn't find an _ieq operator in hasura either, I didnt notice it was in the common operators list we have here 🙈